src/appchooserdialog.c

```diff
@@ -132,6 +132,8 @@ show_more (AppChooserDialog *dialog)
   gtk_window_set_resizable (GTK_WINDOW (dialog), TRUE);
   gtk_widget_hide (dialog->more_row);
 
+  g_return_if_fail (g_strv_length ((char **)dialog->choices) > INITIAL_LIST_SIZE);
```

Comment:
Space after the cast's parens:
`g_return_if_fail (g_strv_length ((char **) dialog->choices) > INITIAL_LIST_SIZE);`

Can you also update the commit message to not mention assertions?

Reply:
g_return is an assertion, but I can reword it, sure.

Reply:
This file doesn't use spaces after casts. Let's change that in a follow-up if you prefer it that way.

Reply:
OK for the spaces after casts. As for the g_return, it's a guard, not an assertion (which would call `abort()` on failure) ;)

Reply:
"Assertion guard?" Point is it's a statement that should never fail except in case of programmer error. Whatever, we know how it works. :)